### ISSUE TITLE
Feature: adding drawing operations

### ISSUE THREAD

**Issue by dilumdesilva** [scorelab/imagelab#28]
You can draw various shapes like **Circle**, **Rectangle**, **Line, Ellipse**, **Polylines**, **Convex**, **Polylines**, **Polylines** on an image using the respective methods of the **org.opencv.imgproc** package.

So, in this issue we expect you to add drawing operations to ImageLab.

Check the [project documentation](https://www.scorelab.org/ImageLab/) for more information related to the project setup and contributions.

**Comment by diegoliber** [comment]
Hi, a little more information about the drawing operators. What parameters do you expect from the users? Do you think it's interesting to put an aid for identifying image coordinates in the preview?

**Comment by dilumdesilva** [comment]
Hey @diegoliber , if we consider something like drawing a circle on top of a given image (in Mat format), the OpenCV circle function would accept something like `circle(img, center, radius, color, thickness)` . So, as the centre, you can pass the coordinates as a point value.

**Comment by diegoliber** [comment]
@dilum1995  Yes, I understand that. Implementing the form and performing the drawing operation by calling OpenCV is simple enough. The thing is, on the perspective of the user, it has to know the exact coordinates he wants to draw from the image. This makes the feature easy to implement, but this kinda hurts the user experience. It's different from applying a blur filter to the whole image. Unless, of course, you don't care about any of this.

**Comment by dilumdesilva** [comment]
Hey, @diegoliber oh now I got your point. Yes, then we have to have some kind of a grid on top of preview pane with coordinates or something like that. Do you have anything in your mind.... would love to hear your version?


And if you're interested feel free to go ahead and send us a PR
